### PR TITLE
fix(ToggleContent): Make ToggleContent look identical to ReadMore

### PR DESCRIPTION
**Pull request by kosanna**
Currently ToggleContent and ReadMore have inconsistent look. This PR makes them look the same.

### CODE REVIEW

File: react/ToggleContent/ToggleContent.less [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import (reference) '~seek-style-guide/theme';
+
+.button {
+  box-shadow: none !important;
```
Comment (georgespyropoulos) [inline react/ToggleContent/ToggleContent.less]: Looks like box shadow has already been set to none. Do we need to declare it again?

screen shot 2018-12-17 at 9 47 19 am

Comment (kosanna) [reply react/ToggleContent/ToggleContent.less]: we need to overwrite focus state coming from the button, I'll just do that directly

Comment (georgespyropoulos) [reply react/ToggleContent/ToggleContent.less]: I'm confused - why would we want to override that style? If we turn off the box shadow there's no way to tell your focussed on the button when using the page with a keyboard

Comment (georgespyropoulos) [reply react/ToggleContent/ToggleContent.less]: But that breaks keyboard accessibility of the button. Why do you want to reset it?

**Pull request closed without merging**